### PR TITLE
Add session.consumer_id, add authorization for session_stats_create

### PR DESCRIPTION
**Pull request by donce**

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 
 NODE_KEY_LIMIT = 42
-SESSION_KEY_LIMIT = 36
+IDENTITY_LENGTH_LIMIT = 36
```
Comment (interro) [inline models.py]: identity length should be 42, not 36

File: app.py [hunk 2]
```diff
@@ -160,7 +160,10 @@ def session_stats_create(session_key, caller_identity):
     if session is None:
         session = Session(session_key)
         session.client_ip = request.remote_addr
+        session.consumer_id = caller_identity
 
+    if session.consumer_id != caller_identity:
+        return jsonify({}), 403
```
Comment (interro) [inline app.py]: missing error message

Comment (donce) [reply app.py]: Added.

File: models.py (hunk 1 above)
Comment (donce) [reply models.py]: Good catch, fixed.

File: app.py [hunk 3]
```diff
@@ -152,15 +152,19 @@ def session_stats_create(session_key, caller_identity):
...
-        return jsonify({'error': 'bytes_sent should not be negative'}), 400
+        return jsonify(error='bytes_sent should not be negative'), 400
     if bytes_received < 0:
-        return jsonify({'error': 'bytes_received should not be negative'}), 400
+        return jsonify(error='bytes_received should not be negative'), 400
 
     session = Session.query.get(session_key)
     if session is None:
         session = Session(session_key)
         session.client_ip = request.remote_addr
```
Comment (Waldz) [inline app.py]: Would be worth to sync names:
- `client_id` + `client_ip`
- `consumer_id` + `consumer_ip`

File: tests/test.py [hunk 4]
```diff
@@ -77,53 +77,77 @@ def test_proposals_with_unknown_node_key(self):
...
+    def test_session_stats_create_successful(self):
+        payload = {
+            'bytes_sent': 20,
+            'bytes_received': 40,
+        }
+        auth = generate_test_authorization(json.dumps(payload))
 
-    def test_session_stats_create_with_session_record(self):
         session = Session('123')
+        session.consumer_id = auth['public_address']
```
Comment (Waldz) [inline tests/test.py]: I notice again that not passing `consumer_id` in payload makes API stateful.

Comment (donce) [reply tests/test.py]: `consumer_id` is passed in url:
```
'/v1/sessions/123/stats',
```
in this case, `consumer_id` is `123`.

File: app.py (hunk 3 above)
Comment (donce) [reply app.py]: Same applies for other model fields as well: `client_bytes_sent`, `client_bytes_received`, `node_bytes_sent`, `node_bytes_received`, `client_updated_at`, `node_updated_at`, `node_key`.
Lets not solve all problems at once.

File: tests/test.py (hunk 4 above)
Comment (Waldz) [reply tests/test.py]: No, it's session unique ID

Comment (donce) [reply tests/test.py]: Yeah you're right.

**Pull request merged**